SwiftSnake/SwiftSnake/Extensions/GameLogicControllers/PieceToDisplay.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  PieceToDisplay.swift
+//  SwiftSnake
+//
+//  Created by Ellen Shapiro (Vokal) on 9/24/15.
+//  Copyright © 2015 Vokal. All rights reserved.
+//
+
+import Foundation
+
+enum PieceToDisplay {
+    case SnakeHead,
```

Comment:
I think enums like this would be easier to quickly scan/read with a line break after the `case` so each case is on its own line.